### PR TITLE
PyData Warsaw

### PR DESCRIPTION
**Pull request by Daniel-at-github**

### CODE REVIEW

**Comment by Daniel-at-github** [comment]
Hi @stared, I take the opportunity with this pull request to contact team members of PyData Warsaw.
Please, contact with them. If anyone on the group is interested in review, add more talks or improve the data in any way, it will be greatly appreciated.

File: pydata-warsaw/category.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "title": "PyData Warsaw"
```
Comment (redapple) [inline pydata-warsaw/category.json]: @Daniel-at-github , could this (and the folder name) also include the year?

Comment (Daniel-at-github) [reply pydata-warsaw/category.json]: This are the normal group reunion videos, not the PyData Warsaw 2017 conference.
I supposed the the two can coexist, but no, they can't. See the error:
~~~
CRITICAL: ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S %z'
~~~
What can we do in this cases? ... change the name?

Comment (Daniel-at-github) [reply pydata-warsaw/category.json]: > @redapple could this (and the folder name) also include the year?
No. PyData Warsaw 2017 conference is out, and is a different thing.

Comment (redapple) [reply pydata-warsaw/category.json]: I would suggest using something like "PyData Warsaw Meetups" and link it to
https://www.meetup.com/PyData-Warsaw/events/

**Pull request merged**